### DISCUSSION TITLE
Status on compiler & bundler

### DISCUSSION THREAD

**Discussion by viktor89** [rome/tools#4302] [General]
The landing page says "March 2023" for compiler & bundler. I can't see any issues, projects or discussions about the state of those on Github. Is it in progress? And is march 2023 realistic?

PS. I really like what you're building here, so my question only comes from excitement to see those features come to life :)

**Comment by unvalley** [comment]
premise: I'm just a contributor, not a core member.

March 2023 isn't realistic. The landing page is updated to "soon". As you know, currently rome has formatter and linter. It's needed some time to start tackling on compiler and bundler.

Let's be patient and wait!

**Comment by kasperstorgaard** [comment]
Hi Victor
If you pop into the discord there's some info around the project as a whole.
But in short, the company Rome Tools ran out of funding, so the core team of last year are no longer working on the project.
Instead the open source contributors are carrying on the work, but mainly with the linter.

_Note: Anyone feel free to correct me, I would love to get some solid information_

**Reply by re-thc** [reply]
Any further info on this? According to the website 4.5 million was raised in 2021 (less than 2 years ago). How many people do you have to hire to have burned that money...

**Reply by ematipico** [reply]
> Any further info on this? According to the website 4.5 million was raised in 2021 (less than 2 years ago). How many people do you have to hire to have burned that money...

Managing a company is entirely different from programming; people are good at some things and bad at others. It's best to judge only if you have the expertise and the context. It's a piece of bad news regardless, but the community behind the open-source project is still there; we just released a new version.

**Reply by nfwyst** [reply]
Its not a good news.
The front-end field needs a leader too much, an all in one tool, but not rome?
